common/config/rush/pnpm-lock.yaml

```diff
@@ -849,34 +849,34 @@ packages:
       integrity: sha512-mky/O83TXmGY39P1H9YbUpjV6l6voRYlufqfFCvel8l1phuy8HRjdWc1rrPuN53ITBJlbyMSV6z3niOySO5pgQ==
   /@types/fs-extra/8.1.1:
     dependencies:
-      '@types/node': 10.17.51
+      '@types/node': 8.10.66
```

Comment:
Is this supposed to go down?

Reply:
My understanding was this was bumped up accidentally by this PR: https://github.com/Azure/azure-sdk-for-js/commit/189bb99b116f66ef9f9a974fc1ede16be7cb9287#diff-5029779360a5dab7df00ff76c5693ce33b68f76d2f433b44a28b333bf33f674a

/cc @ramya-rao-a  @jonathandturner

Reply:
It was fixed in a previous commit I did too: https://github.com/Azure/azure-sdk-for-js/commit/24cbc288b10b8670301d37303db4ead11c3d578f#diff-5029779360a5dab7df00ff76c5693ce33b68f76d2f433b44a28b333bf33f674a

Looks like we've had a spate of folks not reverting the lockfile to master before running `rush update` when there's a conflict?

Reply:
cc @deyaaeldeen

Reply:
I think the following two observations might have to do with this:
- The dependencies for `@types` packages are typically other `@types` packages with the semvar being just "*". For example, if you look at `common/temp/node_modules/@types/fs-extra/package.json`, you will see the following:
```
 "dependencies": {
        "@types/node": "*"
    }
```
this introduces issues such as https://github.com/microsoft/rushstack/issues/750.
- rush uses pnpm's fewer-deps strategy by default as indicated here: https://github.com/microsoft/rushstack/blob/9d175a4401545b437869d33a2a67c29042ceeb97/apps/rush-lib/src/api/RushConfiguration.ts#L374 and we explicitly use that strategy here: https://github.com/Azure/azure-sdk-for-js/blob/b8739325dc9b8d97d396c86705efbd63675b69e3/rush.json#L54. If I understand correctly, this means pnpm will work hard to use the fewest number of packages possible to satisfy the constraints.